### DISCUSSION TITLE
victron smartshunt

### DISCUSSION THREAD

**Discussion by PVDing** [softwarecrash/Victron2MQTT#5] [Q&A]
hi zusammen.
ich habs mal mit dem smartshunt probiert .daten werden alle auf meine datenbank geschrieben  klappt soweit gut. 
aber der SOC wert des akkus zeigt 0.56% statt 56% an .
kann ich das selber berichtigen oder ist die software für den smartshunt garnicht gemacht ?
freu mich auf antwort und hilfe :) :)
gruss

**Comment by softwarecrash** [comment]
Hallo @PVDing das ist ein fehler in der software.
probier bitte mal diese
[Victron2MQTT_d1_mini_1.1.2C-1.zip](https://github.com/softwarecrash/Victron2MQTT/files/13624265/Victron2MQTT_d1_mini_1.1.2C-1.zip)

**Reply by PVDing** [reply]
hey super nun wirds richtig angezeigt .
grosses dankeschön für die schnelle behebung .
bei mir laufen nun das daly bms  2x victron regler und der smartshunt . bin super zufrieden . 
und ein grosses dankeschön für die tolle arbeit an das team . nun hab ich kein kabelgebundenes system am rasberry mehr

**Reply by softwarecrash** [reply]
Super, der fix geht dann mit dem nächsten Bugfix rein.

Freut mich das alles so gut läuft.
